.gitignore

```diff
@@ -3,3 +3,4 @@
 *.pyc
 *.pyo
 __pycache__/*
+basic_subscription
```

Comment:
This is a downstream component; upstream should have no knowledge of downstream. Please remove it.